scripts/script.yaml

```diff
@@ -126,27 +127,29 @@
                         options:
                           - show: "כן"
                             steps:
-                              - say: מה מספר החולה שפגשת?
+                              - say: מצויין. מה המספר?
                               - wait:
                                   variable: insulation_patient_number
                                   placeholder: "מספר החולה שפגשתי"
-                              - say: האם ידוע לך התאריך שבו נפגשתם?
+                              - say: האם זכור לך היום שבו נפגשתם?
```

Comment:
@mushon I think we can use this question only when we have a date picker (#24)  / validation. Otherwise, we will get day names instead of dates.